pxemgr/ipxe_handlers.go

```diff
@@ -105,6 +106,14 @@ func (mgr *pxeManagerT) maybeCreateHost(serial string) *hostmgr.Host {
 			}
 		}
 
+		if mgr.config.Network.NetworkModel == "bridge" || mgr.config.Network.NetworkModel == "bond_bridge" {
+			host.BridgeIP = mgr.getNextBridgeIP()
```

Comment:
Let's not use different ips for the bridges. that won't work ootb anyway. I'd remove this part. You should use flannel or something similar for this.